src/models/codedelegate.cpp

```diff
@@ -12,8 +12,7 @@
 #include <QPainter>
 #include <QTextLine>
 
-#include "disassemblymodel.h"
-#include "sourcecodemodel.h"
+Q_DECLARE_METATYPE(QTextLine)
```

Comment:
how is this related to the copy operation? separate patch?

Reply:
can you please answer the question?

Reply:
clangd was complaining the whole time about unused includes so I removed those